package.json

```diff
@@ -209,6 +209,7 @@
   },
   "dependencies": {
     "node-gyp": "^9.0.0",
+    "soundcloud-scraper": "^5.0.3",
```

Comment:
might be able to ditch this dependancy - all we are using it for is client_id keygen and soundcloud search functionality

Reply:
That client id is crucial - if you can make do without it, feel free to remove it.